php/Terminus/Models/Workflow.php

```diff
@@ -83,18 +73,19 @@ public function wait() {
     while (!$this->isFinished()) {
       $this->fetch();
       sleep(3);
-      print ".";
```

Comment:
Yeah, but to be honest I haven't replaced it with anything all that much better. There really is no other use case for this progress bar thingy and it needs to send output to the console without the timestamps that our logger adds and without ending up in the actual output.

I moved this into it's own static function in Terminus, so at least it's not a totally unaccounted for 'print' but at the end if the day it's doing much the same thing (but to STDERR not STDOUT so as to not pollute output if there is any)